zipline/algorithm.py

```diff
@@ -2374,7 +2374,7 @@ def set_long_only(self, on_error='fail'):
     # Pipeline API
     ##############
     @api_method
-    @require_not_initialized(AttachPipelineAfterInitialize())
+#    @require_not_initialized(AttachPipelineAfterInitialize())
```

Comment:
Could you please remove this line (as opposed to commenting it out)?

Reply:
ok, I didnt know wether you wanted to keep the history in the code